server/application/application.go

```diff
@@ -183,11 +183,7 @@ func (s *Server) Create(ctx context.Context, q *application.ApplicationCreateReq
 	defer s.projectLock.RUnlock(q.Application.Spec.Project)
 
 	a := q.GetApplication()
-	validate := true
-	if q.Validate != nil {
-		validate = *q.Validate
-	}
-	err := s.validateAndNormalizeApp(ctx, a, validate)
+	err := s.validateAndNormalizeApp(ctx, a, q.Validate)
```

Comment:
Validate is another field where omission matters. Omitted actually equates to true.